### ISSUE TITLE
Linking to a company's press kit/guidelines/common errors

### ISSUE THREAD

**Issue by ericcornelissen** [simple-icons/simple-icons#755]
tl;dr: Should we **a)** include a link to either the brand guidelines of an icon or **b)** include a link to a _Common Errors_ section of an icon (if available)

* * *

From #749 we found that it can be somewhat unclear what color should be linked to an icon. From the [ Firebase brand guidelines](https://firebase.google.com/brand-guidelines/) it is clear that the primary color for Firebase is `#FFCA28`, but the [Common Errors](https://firebase.google.com/brand-guidelines/#common-errors) section it is specified that you're not supposed to use the logo in any other color then white (i.e. `#FFFFFF`).

> Do not use the flat knockout in any color but white

It was then decided to use `#FFCA28` on the website since 1) when the SVG is downloaded has no color associated with it and 2) that "is" the brand color of Firebase.

However, it may be an idea to include a link to either the brand guidelines of an icon (for all icons) **or** include a link to a _Common Errors_ section of an icon (if one exists/is known). So that user can check the documentation for themselves when using an icon.

**Comment by ericcornelissen** [comment]
Just throwing out there that we can take some inspiration from [BrandColors](https://brandcolors.net/) which provide a link to the brand URL and the source URL as well as the date it was updated. E.g. for [500px](https://brandcolors.net/b/500px):

![image](https://user-images.githubusercontent.com/3742559/38525641-5d3bcc30-3c53-11e8-9215-c05a8b87d5ac.png)

**Comment by ericcornelissen** [comment]
> It's worth noting that many brands have exceptions to these guidelines. For example:
Brands that stipulate that a solid white icon on a colored background may still use this treatment in the Windows App store, the MacBook touch bar, an app icon, a social media avatar, or other places.

If that's the case, it should be possible to find that in the guidelines, so I see no problem there.


> I'm curious what the importance or value is in following brand guidelines beyond the shape of their mark. Could those guidelines be cited and left to the discretion of the user to style them correctly? I'm not sure many users are utilizing Simple Icons for official branded material.

I won't be surprised if not everyone using SimpleIcons is going to look at the brand guidelines, but some folk might be concerned about this stuff and for them a simple link can help a ton. And since we already have a lot of these links available to us (as the `source` of an icon) I see no reason to just incorporate them to the site. But, as you said, I do think that just citing them and let that be it is enough.
